Sources/Private/MainThread/NodeRenderSystem/Nodes/RenderNodes/StrokeNode.swift

```diff
@@ -118,7 +118,7 @@ final class StrokeNode: AnimatorNode, RenderNode {
 
     /// Get dash lengths
     let dashLengths = strokeProperties.dashPattern.value.map { $0.cgFloatValue }
-    if dashLengths.count > 0 {
+    if dashLengths.count > 0 && !dashLengths.allSatisfy({ $0.isZero }) {
```

Comment:
Same as https://github.com/airbnb/lottie-ios/pull/1691/files#r936164838:

```suggestion
    if dashLengths.count > 0, !dashLengths.allSatisfy({ $0.isZero }) {
```